examples/tfengine/generated/org_foundation/cicd/triggers/apply/variables.tf

```diff
@@ -43,12 +43,19 @@ variable "skip" {
 
 variable "run_on_push" {
   type        = bool
-  description = "Whether or not to be automatically triggered from a PR/push to branch."
+  description = <<EOF
+Whether or not to be automatically triggered from a PR/push to branch.
+Default to true.
```

Comment:
Sure. 

It only make sense if you deploy this trigger standalone, otherwise the variable comes from the root folder, and there can't be a default because it is a nested value.

The default is set on the `.tfvars` via templating.

Added it anyways